pt.el

```diff
@@ -275,7 +280,10 @@
           pt-fillcolor-default
           pt-color-unavailable
           pt-fontcolor-unavailable
-          (apply 'concat (mapcar 'pt-goal->dot goals))))
+          (apply 'concat
+                 (mapcar 'pt-goal->dot
+                         (sort goals
```

Comment:
Also worth noting: `sort` function appears to modify the underlying datastructure, but not to store the sorted list, so... `goals` will be pretty much useless after sorting is done.